.gitignore

```diff
@@ -81,3 +81,6 @@ celerybeat-schedule.db
 # node modules
 **/*/node_modules
 node_modules
+
+# local tmp folder
+tmp
```

Comment:
why is tmp needed?

Reply:
not needed I added so I can keep track of some local files without committing them. I can remove them if you think it is confusing.